### PR TITLE
✨ Add functions to check which player goes first at start of game

### PR DESCRIPTION
**Pull request by fre-ben**
- After every player revealed two cards the current roundScore will determine which player goes first
- Player with the highest roundScore goes first
- ActivePlayer is set in backend gamestate
- ActivePlayer info is being emitted to the frontend

Deployment: https://sisu-pipelin-goesfirst-awqhjev.herokuapp.com/
Fixes parts of #73

### CODE REVIEW

**Review by sweisz — COMMENTED** [review]
GJ ♦️

File: pages/game.tsx [hunk 1]
```diff
@@ -83,16 +90,24 @@ export default function Game() {
...
-  const handleCardGridClick = (index: number) => {
-    if (gameHasStarted) {
+  const handleCardGridClick = (index: number): void => {
+    // example for further functionality
+    // if (activePlayer === player) {
+    //   switch (phase) {
+    //     case 'PICK': pickCard(index)
+    //     break;
+    //   }
+    // }
```
Comment (phlgr) [inline pages/game.tsx:100]: This should be removed if not necessary.

File: pages/game.tsx [hunk 2]
```diff
@@ -127,6 +142,8 @@ export default function Game() {
 
       <main className={styles.viewContainer}>
         <Logo size="small" />
+        {/* Just to check if activePlayer works */}
```
Comment (phlgr) [inline pages/game.tsx:145]: This can be removed

**Review by phlgr — COMMENTED** [review]
Clean Code 😄 
LGTM 🚀

File: pages/game.tsx (hunk 2 above)
Comment (fre-ben) [reply pages/game.tsx:145]: It will be :P

**Pull request merged**